docker/install_packages.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+. /build/install_pecan_preprocessor.sh
+
+echo "######################################################################"
+echo "SETTING UP REPOS"
+echo "######################################################################"
+case "$OS_VERSION" in
```

Comment:
General comment -- Thinking about this from the perspective of a user adding a model, or trying to port PEcAn to a new OS, I wonder if it would be easier if you had a set of different install_packages scripts, one for each OS.

Also, maybe this is just because things are a work in progress, but I assumed that having different dockers for different models would involve those individual pieces being responsible for what model packages needed to be installed (rather than a central script for installing all the packages for all models)